frontend/src/constants.ts

```diff
@@ -7,3 +7,4 @@ export const FRONTEND_URI =
 	import.meta.env.REACT_APP_FRONTEND_URI ||
 	window.location.protocol + '//' + window.location.host
 export const AUTH_MODE = import.meta.env.REACT_APP_AUTH_MODE
+export const DISABLE_ANALYTICS = import.meta.env.REACT_APP_DISABLE_ANALYTICS
```

Comment:
thanks for the changes, now - you'll need to make sure this env var is passed thru from the hobby deployment. could you update the `docker/compose.hobby.yml` and `docker/.env` / `docker/env.sh` accordingly?